PyMPDATA/solver.py

```diff
@@ -36,9 +36,11 @@ def call(self, flux, g_factor, step, iteration):  # pylint: disable=unused-argum
 
 
 class Solver:
-    """solution orchestrator requireing prior instantiation of: a `Stepper`,
-    a scalar advectee field, a vector advector field and optionally
-    a scala g_factor field"""
+    """Solution orchestrator requiring prior instantiation of: a `Stepper`,
+    a scalar advectee field (that is acted upon by the advector),
+    a vector advector field (that acts upon the advectee),
```

Comment:
I wonder if this might be misleading: in the case of a momentum conservation eq., advectee effectively also acts upon the advector (as in the shallow-water examples) - likely worth referring to these examples to make it clear?

Reply:
Added a note about momentum advection and a reference to Randall's book